### PR TITLE
faas-env default qa

### PR DESCRIPTION
**Pull request by seanchoi-dev**
Resolves: 
https://workfront.slack.com/archives/C03B0BUA75E/p1704493717108139

**Test URLs:**
- Before: https://main-fix--milo--adobecom.hlx.page/drafts/seanchoi/faas-link-test?martech=off
- After: https://faas-fix--milo--adobecom.hlx.page/drafts/seanchoi/faas-link-test?martech=off

Note that `qa` environment requires company network or VPN connection.

### CODE REVIEW

**Review by rgclayton — APPROVED** [review]

File: libs/blocks/faas/utils.js [hunk 1]
```diff
@@ -26,12 +26,12 @@ export const getFaasHostSubDomain = (environment) => {
...
     return 'qa.';
   }
-  return 'dev.';
+  return 'qa.';
 };
 
 const base = miloLibs || codeRoot;
-export const faasHostUrl = `https://${getFaasHostSubDomain()}apps.enterprise.adobe.com`;
-const faasCurrentJS = `${faasHostUrl}/faas/service/jquery.faas-current.js`;
+export const faasHostUrl = base.includes('localhost') ? `${base}/deps` : `https://${getFaasHostSubDomain()}apps.enterprise.adobe.com/faas/service`;
```
[the remark this answers is not in the document]
Comment (seanchoi-dev) [reply libs/blocks/faas/utils.js:33]: Yes, it is moved from right above.

[the remark this answers is not in the document]
Comment (meganthecoder) [reply libs/blocks/faas/utils.js:33]: Right, but `faasHostUrl` is being exported, so presumably it's being imported and used somewhere other than this file. Seems like we should check and make sure that the changes to `faasHostUrl` don't break functionality in the file where it's being imported.

[the remark this answers is not in the document]
Comment (meganthecoder) [reply libs/blocks/faas/utils.js:33]: Did a search and found it here in the faas config https://github.com/adobecom/milo/blob/3ba3c1be0a28d14324b57a2714dd9c48c3c3b9dd/libs/blocks/faas-config/faas-config.js#L81

[the remark this answers is not in the document]
Comment (seanchoi-dev) [reply libs/blocks/faas/utils.js:33]: You're right, I will fix it.

File: libs/blocks/faas/utils.js [hunk 2]
```diff
@@ -26,12 +26,12 @@ export const getFaasHostSubDomain = (environment) => {
...
     return 'qa.';
   }
-  return 'dev.';
+  return 'qa.';
 };
 
 const base = miloLibs || codeRoot;
 export const faasHostUrl = `https://${getFaasHostSubDomain()}apps.enterprise.adobe.com`;
-const faasCurrentJS = `${faasHostUrl}/faas/service/jquery.faas-current.js`;
+const faasCurrentJS = base.includes('localhost') ? `${base}/deps/jquery.faas-current.js` : `${faasHostUrl}/jquery.faas-current.js`;
```
Comment (meganthecoder) [inline libs/blocks/faas/utils.js:34]: Getting a 404 error in the console for `https://qa.apps.enterprise.adobe.com/jquery.faas-current.js `. Looks like you accidentally dropped the `/faas/service` part

Comment (seanchoi-dev) [reply libs/blocks/faas/utils.js]: I am so sorry, I was doing multiple things at the moment and didn't do the proper test myself. This surely works, thanks for caching all the misses.

**Review by meganthecoder — APPROVED** [review]

File: libs/blocks/faas/utils.js (hunk 2 above)
Comment (meganthecoder) [reply libs/blocks/faas/utils.js]: No problem, looks good now

**Pull request merged**